js/toast.js

```diff
@@ -0,0 +1,14 @@
+let toastO=document.getElementById('toastO');
+let toastI=document.getElementById('toastI');
+
+let toastMessage=(message,backgroundColor)=>{
```

Comment:
required paramters should be message and toast display time. rest should be optional